### DISCUSSION TITLE
ServiceStack Blazor Template not running anymore.

### DISCUSSION THREAD

**Discussion by Pinox** [ServiceStack/Discuss#112] [Q&A] [answered]
Hi Guys,

Just to let you know that SS WASM template no longer works as expected.

I tend to select: 
![image](https://github.com/ServiceStack/Discuss/assets/5297255/9ec2ac10-090e-4f0d-9d93-6db209ca0f94)

Would be great if you can also update the templates to add nugets when using SqlServer
`PackageReference Include="ServiceStack.OrmLite.SqlServer" Version="8.*" `
`PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="8.0.2" ` 

and then in Program.cs:

`services.AddDbContext<ApplicationDbContext>(options =>
     options.UseSqlServer(connectionString, b => b.MigrationsAssembly(nameof(MyApp))));` 

When I try and run then I get the following error:

![image](https://github.com/ServiceStack/Discuss/assets/5297255/f591cada-4efa-4fd4-aa4b-e095b16ca802)

Also tested this on Sqlite version , same result.

**Comment by mythz** [comment]
Switching to a different database should now be improved.

**Comment by Pinox** [comment]
Thanks @mythz .

I notice Sqlite version now have code not seen before : 

`using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;` 

`ApplicationDbContextModelSnapshot.cs`


SqlServer version runs only if I uncomment file TodosServices.cs:  

`public class TodosServices(IAutoQueryData autoQuery) : Service`

So if I do not uncomment above file then I still get error trying to run as in my original post.

Anything we need todo to get this (TodoService) running again ?

**Reply by mythz** [reply, marked as answer]
You shouldn't select AutoQuery for templates which already have AutoQuery as that would overrides the more appropriate modified templates version.

I've fixed SQLIte, but you shouldn't use that either as all Identity Auth + AutoQuery templates are configured with SQLite by default.
